examples/gluon_alexnet/gluon_alexnet.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import mxnet
+from mxnet import gluon
+from mxnet.gluon import nn
+from mms.model_service.mxnet_vision_service import MXNetVisionService
+import numpy as np
+
+"""
+MMS examples for loading Gluon Hybrid/Dynamic models
+"""
+
+
+class ImperativeAlexNet(gluon.Block):
+    """
+    Fully imperative gluon Alexnet model
+    """
+    def __init__(self, classes=1000, **kwargs):
+        super(ImperativeAlexNet, self).__init__(**kwargs)
+        with self.name_scope():
+            self.features = nn.Sequential(prefix='')
```

Comment:
Why this complex implementation of Alexnet? Why do you need both features and dense? you can use just one net that has all of them.